flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+  "only-arches": [
+    "x86_64"
+  ],
+  "automerge-flathubbot-prs": true
```

Comment:
```suggestion
  ]
```

Reply:
I would prefer to have the flathubbot PRs get merged automatically